infra/app.js

```diff
@@ -46,14 +46,16 @@ async function getBuckets() {
 
   const deployStart = Date.now();
   console.info(`Deploying build: ${buildNumber} of wellcomecollection.org to prod...`);
-  
-  exec(`./deploy.sh ${bucket} ${buildNumber}`, (err, stdout) => {
-    if (err) {
-      console.error('There was an error deploying, please make sure you check that the AWS setup is not mangled.');
+
+  exec(`./deploy.sh ${bucket} ${buildNumber}`, (err, stdout, stderr) => {
+    if (stderr) {
```

Comment:
It should indeed.
[Done](https://github.com/wellcometrust/wellcomecollection.org/pull/66/commits/ba041b2e659dbf0343025a243961e5ff91ddf8a6).